src/const/meta.ts

```diff
@@ -22,6 +22,7 @@ export const CONFIG = {
     cowProtocol: "https://cow.fi",
     beaver: "https://beaverbuild.org/",
     agnostic: "https://agnostic-relay.net/",
+    cookiePolicy: '/cookie-policy'
```

Comment:
Mixed single/double quotes, I think we should use only one

Reply:
Good catch 👍